src/e2e-tests/page-objects/autelis.page.js

```diff
@@ -1,11 +1,211 @@
 // autelis.page.js
 import Page from './page';
 import MenuComponent from './menu.component';
+let counter = 0;
 
 class AutelisPage extends Page {
+    get pageContent() {return $('//div[contains(@class, "ion-page")]')}
+    get autelisTab() {return $('#autelis-tabs-tab-autelis')}
+    get offButton() {return $('//button[text()[contains(., "OFF")]]')}
+    get poolButton() {return $('//button[text()[contains(., "POOL")]]')}
+    get spaButton() {return $('//button[text()[contains(., "SPA")]]')}
+    get switchMessageMain() {return $('#autelis-tabs-tabpane-autelis>div>div:first-of-type>div:nth-of-type(2)>div:nth-of-type(2)')}
+    get switchMessageSolar() {return $('#autelis-tabs-tabpane-autelis>div>div:first-of-type>div:nth-of-type(3)>div:nth-of-type(2)')}
 
-    get pageContent() { return $('//div[contains(@class, "ion-page")]') }
+    get solarOnButton() {return $('#autelis-tabs-tabpane-autelis>div>div>div:nth-of-type(3)>div>button:first-of-type')}
+    get solarOffButton() {return $('#autelis-tabs-tabpane-autelis>div>div>div:nth-of-type(3)>div>button:last-of-type')}
+    get cleanerOnButton() {return $('#autelis-tabs-tabpane-autelis>div>div>div:nth-of-type(4)>div>button:first-of-type')}
```

Comment:
A lighter selector here can be: `//*[@id="autelis-tabs-tabpane-autelis"]//div[text()="Cleaner"]/parent::*//button[text()="On"]`
Please don't rely on element positions, they can change even on a framework version upgrade.

This is a general situation

Reply:
Updated

Reply:
Maybe you forgot to push? the locators are the same as before